webapp-django/crashstats/crashstats/models.py

```diff
@@ -418,12 +418,14 @@ class SocorroMiddleware(SocorroCommon):
 
     # By default, no binary response
     API_BINARY_RESPONSE = {}
-    # and thus no needed binary filename
-    API_BINARY_FILENAME = None
 
     # By default no special permissions are needed for binary response
     API_BINARY_PERMISSIONS = ()
 
+    @classmethod
+    def get_binary_filename(cls, params):
+        return None
```

Comment:
I made this a class method. It's only used by RawCrash and it's used because the various dumpy things all have different names. Making it a class method allows us to return a differently shaped name depending on what's being requested.